### DISCUSSION TITLE
AttributeError: 'LuceneImpactSearcher' object has no attribute 'set_language'

### DISCUSSION THREAD

**Discussion by Wenze7** [castorini/pyserini#1365] [Q&A] [answered]
When i set "--language zh" in searching , there is an error "AttributeError: 'LuceneImpactSearcher' object has no attribute 'set_language'" 
And i follow the following notebook to search.
https://github.com/castorini/pyserini/blob/master/docs/experiments-spladev2.md
I want to ask if i can't use splade to process Chinese corpus?

**Comment by lintool** [comment]
Unfortunately, there is currently on SPLADE model for Chinese.

**Reply by Wenze7** [reply]
Thanks!
And I want to know what the diff of Chinese and English when using splade?
I can pre-process the tokens and weights for splade, maybe can i not to set language?

**Comment by MXueguang** [comment, marked as answer]
The original version of splade is only trained for English using `bert-base-uncased` model.
If you already trained your Chinese version of splade with model like `bert-base-multilingual-cased` and finished encoding, you can directly follow the document to do indexing and search, without the need of setting `--language zh`.

**Reply by Wenze7** [reply]
Thanks!
